server/src/main/scala/com/emaginalabs/kuronometer/server/config/KuronometerJacksonModule.scala

```diff
@@ -0,0 +1,35 @@
+package com.emaginalabs.kuronometer.server.config
+
+import com.emaginalabs.kuronometer.core.model.{BuildTool, Platform}
+import com.fasterxml.jackson.core.{JsonGenerator, JsonParser}
+import com.fasterxml.jackson.databind._
+import com.twitter.finatra.json.modules.FinatraJacksonModule
+
+object KuronometerJacksonModule extends FinatraJacksonModule {
+  override protected val propertyNamingStrategy: PropertyNamingStrategy = PropertyNamingStrategy.LOWER_CAMEL_CASE
+
+  override val additionalJacksonModules = Seq(
+    new com.fasterxml.jackson.databind.module.SimpleModule {
+      addDeserializer(classOf[Platform], PlatformDeserializer)
+      addSerializer(classOf[Platform], PlatformSerializer)
+
+      addDeserializer(classOf[BuildTool], BuildToolDeserializer)
+      addSerializer(classOf[BuildTool], BuildToolSerializer)
+    })
```

Comment:
Why don't we use [circe](https://github.com/circe/circe) or [lift-json](https://github.com/lift/lift/tree/master/framework/lift-base/lift-json)? Seems easier to configure!